doc/release-notes/0.9/0.9.md

```diff
@@ -0,0 +1,184 @@
+
+
+## Notable changes in this release
+
+The following table covers notable changes in v0.9.0. Further information about these changes can be found in the [user documentation](https://www.eclipse.org/openj9/docs/version0.9/).
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1288">#1288</a></td>
+<td valign="top">SharedClassCacheInfo no longer contains a JVMLEVEL constant</td>
+<td valign="top">OpenJDK10 and later: All platforms</td>
+<td valign="top">SharedClassCacheInfo.getCacheJVMLevel() used to return the JVMLEVEL constant that maps to a Java version number, for example <code>JVMLEVEL_JAVA8</code>. This call now returns only the Java version number, for example <code>10</code> for Java 10.</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/378">#378</a></td>
+<td valign="top">DDR support is now available</td>
+<td valign="top">Linux, Windows</td>
+<td valign="top">Ability to diagnose problems with the VM, garbage collector, or JIT.</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1699">#1699</a></td>
+<td valign="top">Specify heap limits as a percentage of total memory</td>
+<td valign="top">All platforms</td>
+<td valign="top">Ability to easily set heap limits with the <b>-XX:MaxRAMPercentage</b> and <b>-XX:InitialRAMPercentage</b> options instead of statically setting them with <b>-Xmx/-Xms</b> options.</td>
+</tr>
+<!-- Remove? Confirm.<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1776">#1776</a></td>
+<td valign="top">Extra page of memory no longer required when specifying an object heap size that is a multiple of the page size.</td>
+<td valign="top">Linux on Z and z/OS</td>
+<td valign="top">This change avoids wasting memory or calculating a heap size that is 16 bytes less than the page size.</td>
+</tr>-->
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1668">#1668</a></td>
+<td valign="top">Idle tuning feature enabled.</td>
+<td valign="top">Linux on Z, Linux on Power systems</td>
+<td valign="top">Ability to maintain a minimal memory footprint during the life time of an application.</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1517">#1517</a></td>
+<td valign="top">Changes to the default Java heap size for applications that run inside a container.</td>
+<td valign="top">Linux</td>
+<td valign="top">By default, 75% of the physical memory is allocated for the maximum Java heap setting when the <code>-XX:+UseContainerSupport</code> option is set.</td>
```

Comment:
Please refer to the feature doc in the User Guide.

Reply:
The intro to the notable changes table points to the User guide "What's new" for more information. In that topic I will add a link to the -XX option. See https://github.com/SueChaplain/openj9-docs/blob/99e1ff3d699736231962dcc88cdb0c7aa7be6343/docs/version0.9.md